net-dns/maradns/maradns-2.0.14.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd toolchain-funcs user
+
+DEADWOOD_VER="3.2.10"
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="http://www.maradns.org/"
+SRC_URI="http://www.maradns.org/download/${PV%.*}/${PV}/${P}.tar.bz2"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="authonly examples ipv6"
+
+DEPEND=""
+RDEPEND=""
+
+pkg_setup() {
+	ebegin "Creating group and users"
+	enewgroup maradns 99
+	enewuser duende 66 -1 -1 maradns
+	enewuser maradns 99 -1 -1 maradns
+	eend ${?}
+}
+
+src_prepare() {
+	default
+	# Use duende-ng.c.
+	cp  "${S}/tools/duende-ng.c" "${S}/tools/duende.c" \
+		|| die "failed to rename duende-ng.c"
+	# replace "make" by "$(MAKE)" to avoid GNU Make warning
+	sed -i 's/\ make\ /\ \$\(MAKE\)\ /g' "${S}/build/Makefile.linux" \
+		|| die "can't patch build/Makefile.linux"
+}
+
+src_configure() {
+	tc-export CC
+	./configure $(use ipv6 && echo "--ipv6") || die "Failed to configure ${PN}"
+}
+
+src_install() {
+	# Install the MaraDNS and Deadwood binaries
+	dosbin server/maradns
+	dosbin tcp/zoneserver
+	if ! use authonly; then
+		dosbin deadwood-${DEADWOOD_VER}/src/Deadwood
+	fi
+	dobin tcp/getzone tcp/fetchzone
+	dobin tools/askmara tools/askmara-tcp tools/duende
+	dobin tools/bind2csv2.py tools/csv1tocsv2.pl
```

Comment:
`dobin tools/{askmara,askmara-tcp,duende,bind2csv2.py,csv1tocsv2.pl}`
and `.py` and `.pl` scripts dont need python and perl support at runtime?